test/distribution/test_flows.py

```diff
@@ -69,10 +69,10 @@ def test_flow_invertibility(
     y_hat = bijection.f(bijection.f_inv(inp))
     x_hat = bijection.f_inv(bijection.f(inp))
     assert allclose(
-        inp, y_hat
+        inp, y_hat, rtol=1.0e-4
     ), f"y and y_hat did not match: y = {inp}, y_hat = {y_hat}"
     assert allclose(
-        inp, x_hat
+        inp, x_hat, rtol=1.0e-4
     ), f"y and y_hat did not match: x = {inp}, x_hat = {x_hat}"
```

Comment:
Just realized there’s a typo here in the error string, should be x and x_hat

Reply:
Good catch! Is fixed